ddagent-install.log

```diff
@@ -0,0 +1,295 @@
+[34m
```

Comment:
I don't think this file should be added

Reply:
@dannyroberts , `pc.pub` is my my working laptop.